### DISCUSSION TITLE
763

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#931] [Comments]
# 763

팀원들의 이력서를 글 하단에 첨부해두었다. 이 사이드 프로젝트를 진행한 멤버들에게 관심이 생긴다면 한번 커피챗을 요청해보자. 올해 대학생분들을 멘토링을 종종 했다. 자주 받던 고민이 "어떻게 해야 취업 경쟁력이 생기느냐" 인데, 나는 그럴때마다 아래와 같이 조언을 드렸다. "뭐든 괜찮으니 본인에게 필요한 것을 하나 만들고 실제 출시까지 해보라. CLI, 패키지, 웹, 앱 상관없이 출시를 하고, 100, 1000명의 고객을 만들고, 365/24시간 운영해보면 로그 메세지는 어떻게 남겨야하고, 에러 핸들링은 어떻게 해야하고, 쿼리는 어떻게 작성해야하고, SQL 인잭션, XSS 공격 등을 왜 막아야하고, 한정된 서버 자원을 어떻게 활용해야 하는지 등등을 배우게 된다. 운영하면서 배운 내용을 블로그에 정리하면 ..

https://jojoldu.tistory.com/763

**Comment by whxogus215** [comment]
안녕하세요, 개발공부 중인 대학교 3학년 학생입니다! 최근 우테코 프리코스에 불합격하면서 무기력함에 빠져있었는데 향로님의 이야기가 많은 도움이 된 것 같습니다. 학교생활과 프리코스 진행 때문에 중단되었던 팀 프로젝트가 있었는데 이번에 다시 진행해서 배포까지 하는 걸 목표로 하려 합니다. 서비스를 운영하면서 겪을 수 있는 다양한 문제들을 블로그에 기록하면서 성장하겠습니다!

**Reply by jojoldu** [reply]
응원합니다 @whxogus215 님 :)

**Comment by SUNGMYEONGGI** [comment]
1000명 유저가 있으면 이미... 취업 포폴이 아니라 창업성공한 앱 아닌가요.

**Reply by jojoldu** [reply]
ㅋㅋㅋ 그럴수도 있겠지만 실제로 1천명 모으는걸로 성공한 창업 앱인 경우는 별로 없는것 같아요.
저는 최근에 소프트웨어 마에스트로 수료식을 참여했는데,
프로젝트 대부분이 다운로드 1만 ~ 3만까지의 앱 혹은 사용자 1천명 혹은 그 이상의 서비스들을 대부분 출시하셨더라구요.
회원수 1천명으로 창업이 성공하기 보다는 
유료 결제 회원수가 1천명은 되어야 성공한 창업이라고 할 수 있을것 같다고 생각합니다 :)

**Comment by sonnbeom** [comment]
안녕하세요! 
향로님 글을 보고 직접 서비스를 운영하는 것을 목표로 잡고 프로젝트를 진행하고 있습니다.
만약 유저 없을 경우에는 글에서 언급해주신 이점을 많이 얻지 못할 것 같은데 이러한 상황에서 좋은 방법이 있을까요?
늘 글을 보고 많은 도움 받고 갑니다!
